drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/indexing/DecisionTableXLSFileIndexer.java

```diff
@@ -20,9 +20,11 @@
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
-
```

Comment:
Please do not remove this empty line, it is required by our codestyle: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration/intellij-configuration/code-style